tasktiger/cli.py

```diff
@@ -0,0 +1,235 @@
+"""TaskTiger CLI."""
+
+import argparse
```

Comment:
We're already using `click` for the `run_worker` CLI, so we should use that instead of `argparse`.